selfdrive/car/tests/test_models.py

```diff
@@ -45,12 +46,17 @@
 SKIP_ENV_VAR = "SKIP_LONG_TESTS"
 
 
-@parameterized_class(('car_model', 'test_route'), test_cases)
-class TestCarModel(unittest.TestCase):
+class TestCarModelBase(unittest.TestCase):
+  car_model = None
+  test_route = None
+  ci = True
 
   @unittest.skipIf(SKIP_ENV_VAR in os.environ, f"Long running test skipped. Unset {SKIP_ENV_VAR} to run")
   @classmethod
   def setUpClass(cls):
+    if cls.__name__ == 'TestCarModel' or cls.__name__.endswith('Base'):
```

Comment:
Skips if base class, and if subclass with default values for car_model, test_route, etc.